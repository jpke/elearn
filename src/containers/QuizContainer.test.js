/* eslint-disable no-undef */
/* eslint-disable import/no-named-as-default */

import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import configureStore from '../store/configureStore';
const store = configureStore();

import QuizContainer from './QuizContainer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <QuizContainer />
    </Provider>
  ).toJSON();
  expect(tree).toMatchSnapshot();
})
