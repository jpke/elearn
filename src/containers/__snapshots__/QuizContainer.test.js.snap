exports[`test renders correctly 1`] = `
<div
  className="quizInProgress">
  <h3
    className="title">
    new quiz
  </h3>
  <h3>
    Question 
    1
  </h3>
  <p>
    no current question
  </p>
  <h3
    className="answers">
    Answers
  </h3>
  <ul
    className="item-list">
    <div>
      <li
        className="none"
        id={undefined}
        onClick={[Function]}>
        no current answer
      </li>
      <li
        className="none"
        id={undefined}
        onClick={[Function]}>
        no current answer true
      </li>
    </div>
  </ul>
  <div
    className="quizNav-buttons">
    <button
      onClick={[Function]}>
      Next Question
    </button>
  </div>
</div>
`;
